### ISSUE TITLE
acc not match(72.03 vs 75.4)

### ISSUE THREAD

**Issue by HappyAIWalker** [Visual-Attention-Network/VAN-Classification#2]
It's same that the release model is not the best one.
python validate.py "MYDATA/ImageNet" --model "van_tiny" --checkpoint "weights/van_tiny_754.pth.tar" -b 200 
The result is following: Acc@1 72.032 (27.968) Acc@5 91.126 (8.874)
![image](https://user-images.githubusercontent.com/85021995/155494990-b54ff097-3881-4600-9cbc-14af27e8b6bb.png)

I check the state_dict of checkpoint and found that ema version is not exist, and I think the ema version's accuracy is same with paper.
![image](https://user-images.githubusercontent.com/85021995/155495361-7b69f29a-527a-400d-80ab-45ee60ec9fa6.png)

so will you check it ?

**Comment by MenghaoGuo** [comment]
Amazing. 

I download the model from google drive and can get the 75.4 % Acc. 

<img width="891" alt="test_tiny" src="https://user-images.githubusercontent.com/38318671/155500734-a3b62346-736c-4454-a504-0af302d04ea3.png">

Here is my eval command

![image](https://user-images.githubusercontent.com/38318671/155501379-9ae13137-b022-406f-8a77-f557b82a8fc5.png)

I also load the model and print key. It is same with you.

![image](https://user-images.githubusercontent.com/38318671/155501778-73019c98-5a97-43bc-ac68-dde12ddce2d3.png)

**Comment by HappyAIWalker** [comment]
it's so strange！ I tried the tiny and small, all not match. I also tried other model (vip and volo) and can got same acc as paper.
the model i tried was download from tsinghua cloud, i will try another one from google cloud later.

**Comment by MenghaoGuo** [comment]
I try to use the model downloaded from tsinghua cloud. However, i also get the 75.4 % acc.  

![image](https://user-images.githubusercontent.com/38318671/155526590-64a7b560-66b3-4ae7-8bd9-d4c373715987.png)

The version of my softwares:

Pytorch == 1.7.1 (I heard it is important)
Timm == 0.4.12

**Comment by HappyAIWalker** [comment]
I try it again and find that the version of timm influence so much.
when version of timm is 0.4.9, the acc is 72.03; when I change it the latest one, the acc is 75.376(matched with your result).
so, maybe there are some difference in the preprocessing part.

**Comment by thunder95** [comment]
I have the same problem, with Pytorch == 1.7.1 and Timm == 0.4.12

Loaded state_dict from checkpoint 'van_tiny_754.pth.tar'
Model van_tiny created, param count: 4105800
Data processing configuration for current model + dataset:
	input_size: (3, 224, 224)
	interpolation: bicubic
	mean: (0.5, 0.5, 0.5)
	std: (0.5, 0.5, 0.5)
	crop_pct: 0.9
Test: [   0/20]  Time: 1.760s (1.760s,  113.65/s)  Loss:  0.8368 (0.8368)  Acc@1:  82.500 ( 82.500)  Acc@5:  98.500 ( 98.500)
Test: [  10/20]  Time: 0.253s (0.469s,  426.49/s)  Loss:  1.1970 (1.1323)  Acc@1:  72.500 ( 76.636)  Acc@5:  93.500 ( 94.364)
 * Acc@1 74.586 (25.414) Acc@5 92.786 (7.214)
